### ISSUE TITLE
Release cli 2.63.0 to MAR

### ISSUE THREAD

**Issue by paulbeattie** [Azure/azure-cli#29658]
### Describe the bug

The July 2024 milestone (2.63.0) of Azure CLI is build complete and [released](https://github.com/Azure/azure-cli/releases/tag/azure-cli-2.63.0).

This should be built and released to [Microsoft Artefact Registry](https://mcr.microsoft.com/en-us/product/azure-cli/tags).

### Related command

docker pull mcr.microsoft.com/azure-cli:2.63.0

### Errors

docker pull mcr.microsoft.com/azure-cli:2.63.0

What's next:
    View a summary of image vulnerabilities and recommendations → docker scout quickview mcr.microsoft.com/azure-cli:2.63.0
Error response from daemon: manifest for mcr.microsoft.com/azure-cli:2.63.0 not found: manifest unknown: manifest tagged by "2.63.0" is not found

### Issue script & Debug output

N/A

### Expected behavior

 Container image would be pulled successfully.

### Environment Summary

N/A

**Comment by jeffpppoint** [comment]
I just started to get errors using github actions and hosted runners using the azure/cli step and i think it's related. In a previous step we download and install azure-cli (2.63.0-1~jammy). When we get to the azure/cli@v2 step we now get 

 Error: Please enter a valid azure cli version. 
See available versions: https://github.com/Azure/azure-cli/releases.
Error: Error: Please enter a valid azure cli version. 
See available versions: https://github.com/Azure/azure-cli/releases.
Warning: Error: EISDIR: illegal operation on a directory, unlink '/home/runner/_work/_temp'
cleaning up container...
Warning: Error response from daemon: No such container: MICROSOFT_AZURE_CLI_1722955565962_CONTAINER

Error: Please enter a valid azure cli version. 
See available versions: https://github.com/Azure/azure-cli/releases.

**Comment by TCFMG** [comment]
We are experiencing the same error, first started occurring `Tue, 06 Aug 2024 04:22:38 GMT` with last successful run at `Tue, 06 Aug 2024 04:09:30 GMT`:

```
Run azure/CLI@v2
Error: Please enter a valid azure cli version. 
See available versions: https://github.com/Azure/azure-cli/releases.

Error: Error: Please enter a valid azure cli version. 
See available versions: https://github.com/Azure/azure-cli/releases.

Warning: Error: EISDIR: illegal operation on a directory, unlink '/home/devuser/actions-runner-1/_work/_temp'
cleaning up container...

Warning: Error response from daemon: No such container: MICROSOFT_AZURE_CLI_1722931206670_CONTAINER

Error: Please enter a valid azure cli version. 
See available versions: https://github.com/Azure/azure-cli/releases.
```
Github action code being used is:

```YAML
- name: Logout of Azure
  uses: azure/CLI@v2
  with:
    inlineScript: |
      az logout
      az cache purge
      az account clear
```

**Comment by bebound** [comment]
2.63 Alpine image is not released yet, see #29628

For GitHub action error, set `azcliversion` to 2.62.0 as a workaround.  @MoChilia for awareness.

**Comment by TCFMG** [comment]
> 2.63 Alpine image is not released yet, see #29628
> 
> For GitHub action error, set `azcliversion` to 2.62.0 as a workaround. @MoChilia for awareness.

Thanks. Have confirmed that using v2.62.0 instead allows the action to succeed:

```YAML
- name: Logout of Azure
    uses: azure/CLI@v2
    with:
    azcliversion: 2.62.0
    inlineScript: |
        az logout
        az cache purge
        az account clear

```

**Comment by MoChilia** [comment]
Hi @paulbeattie, @jeffpppoint, @KurtSunday, @TCFMG, the `azure-cli: 2.63.0` image has been released, see https://mcr.microsoft.com/en-us/product/azure-cli/tags. It is now available for use in `Azure/cli` action.
